src/index.css

```diff
@@ -121,6 +121,16 @@ body {
   }
 }
 
+.badgeBlockName {
+  display: -webkit-box;
```

Comment:
If we can switch to `flex` model, this would be great. If we can't — then let's use this class 👍